lib/views/frontend/spree/checkout/payment/_paypal_braintree.html.erb

```diff
@@ -2,6 +2,10 @@
 <% id = payment_method.id %>
 
 <% content_for :head do %>
+  <script>
+    window.payPalFlow = '<%= SolidusPaypalBraintree::Gateway.first.preferred_paypal_flow %>';
```

Comment:
Are we sure that this global variable is available if we render the button in the cart?